test/constants/json-schemas.js

```diff
@@ -0,0 +1,358 @@
+const vaultsPattern = {
+  type: 'array',
+  items: {
+    type: 'object',
+    properties: {
+      data: {
+        type: 'object',
+        properties: {
+          chain: {
+            type: 'number',
+            minimum: 0,
+            maximum: 1000,
+          },
+          category: {
+            anyOf: [
+              {
+                type: 'string',
+                pattern:
+                  '^(INACTIVE|SUSHI|STABLECOINS|BTC|GENERAL|LIQUIDITY|ONEINCH|SEIGNIORAGE|MSTONKS|ETH20|SUSHI_HODL|LINK|VENUS|PANCAKE|GOOSE|BDOLLAR|NFT|SPACE|INACTIVE_BSC|ELLIPSIS|BELT|UNIV3|LIFT|BALANCER|JARVIS|AMPLIFARM|QUICKSWAP|IDLE|INACTIVE_POLYGON|GENOMES|MSTABLE|POPSICLE|YEL)$',
```

Comment:
rather than hard-coding let's build lists like this dynamically by referencing the corresponding JS files. Updating these lists manually would be difficult each time.